docs/includes/ksql-includes.rst

```diff
@@ -347,3 +435,443 @@ To enable JMX metrics, set ``JMX_PORT`` before starting the KSQL server:
                .. code:: bash
 
                     $ LOG_DIR=./ksql_logs <path-to-confluent>/bin/ksql
+
+.. log_limitations_end
+
+.. __struct_support_01_start
+
+Using Nested Schemas (STRUCT) in KSQL
+=====================================
+
+Struct support enables the modeling and access of nested data in Kafka
+topics, from both JSON and Avro.
+
+Here we’ll use the ``ksql-datagen`` tool to create some sample data
+which includes a nested ``address`` field.
+
+.. __struct_support_01_end
+
+.. __struct_support_02_start
+
+Register the topic in KSQL:
+
+.. code:: sql
+
+    ksql> CREATE STREAM ORDERS WITH (KAFKA_TOPIC='orders', VALUE_FORMAT='AVRO');
+
+Your output should resemble:
+
+.. code:: bash
+
+     Message
+    ----------------
+     Stream created
+    ----------------
+
+Use the ``DESCRIBE`` function to observe the schema, which includes a
+``STRUCT``:
+
+.. code:: sql
+
+    ksql> DESCRIBE ORDERS;
+
+Your output should resemble:
+
+.. code:: bash
+
+    Name                 : ORDERS
+     Field      | Type
+    ----------------------------------------------------------------------------------
+     ROWTIME    | BIGINT           (system)
+     ROWKEY     | VARCHAR(STRING)  (system)
+     ORDERTIME  | BIGINT
+     ORDERID    | INTEGER
+     ITEMID     | VARCHAR(STRING)
+     ORDERUNITS | DOUBLE
+     ADDRESS    | STRUCT<CITY VARCHAR(STRING), STATE VARCHAR(STRING), ZIPCODE BIGINT>
+    ----------------------------------------------------------------------------------
+    For runtime statistics and query details run: DESCRIBE EXTENDED <Stream,Table>;
+    ksql>
+
+Query the data, using ``→`` notation to access the Struct contents:
```

Comment:
We should use the actual ``->`` notation here instead of the arrow character.

Reply:
Good catch. My text editor being too smart!